src/components/Restoration/BusinessName.vue

```diff
@@ -223,9 +212,10 @@ export default class BusinessName extends Vue {
     return this.toDisplayPhone(this.nrApplicant.phoneNumber)
   }
 
-  /** The current options for change of name correction or edit. */
-  get nameChangeOptions (): Array<NameChangeOptions> {
+  /** The current options for name changes. */
+  get correctionNameChoices (): Array<NameChangeOptions> {
     return [
+      // NameChangeOptions.CORRECT_NAME,
```

Comment:
I think I'll leave this option here and commented out -- if we ever reuse this component in Create UI again then we can revisit this.